lib/src/main/java/com/auth0/android/lock/views/PasswordStrengthView.java

```diff
@@ -181,9 +186,20 @@ private int getMinimumLength() {
      * Sets the current level of Strength that this widget is going to validate.
      *
      * @param strength the required strength level.
+     * @deprecated use the {@link #setPasswordComplexity(PasswordComplexity)} method
```

Comment:
Separate PR for deprecations?

Reply:
Seems an overkill since people shouldn't be using our view classes directly.